test/integration/fixtures/TestInstance_withPG.yaml

```diff
@@ -317,7 +317,7 @@ interactions:
     code: 200
```

Comment:
These fixtures will need to be re-ran before merging, I just manually modified the request & response bodies for testing purposes.